src/main/java/com/textmessenger/model/entity/dto/PostToFront.java

```diff
@@ -0,0 +1,49 @@
+package com.textmessenger.model.entity.dto;
+
+import com.textmessenger.model.entity.Post;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class PostToFront {
+  private long id;
+  private String content;
+  private Long parentId;
+  private UserToFrontShort user;
+  private List<UserToFrontShort> likers = new ArrayList<>();
+  private List<CommentToFront> comments = new ArrayList<>();
+  private LocalDateTime createdDate;
+  private LocalDateTime updatedDate;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "updatedDate" private field. ([squid:S1068](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_a9AJjEPbGfqJZL&open=AWWU5_a9AJjEPbGfqJZL)

[ISSUE_KEY]: AWWU5_a9AJjEPbGfqJZL